doc/architecture/decisions/003-authoring-schema-variants.md

```diff
@@ -0,0 +1,41 @@
+# 3. Generate Schema variants from a data source + manifest
+
+Date: 04/07/2017
+
+## Status
+
+Proposed
```

Comment:
I've put a wider list of people to on review for this and as not all of them heard us say that I didn't want it to look like I wasn't open to feedback...